std/bitmanip.d

```diff
@@ -1233,8 +1235,8 @@ public:
     void opSliceAssign(bool val, size_t start, size_t end)
     in
     {
-        assert(start <= end);
-        assert(end <= length);
+        assert(start <= end, "start must not be greater than end");
+        assert(end <= length, "end must be less or equal to the length");
```

Comment:
Why the different wording for essentially the same assertion?

Reply:
fixed

Reply:
fixed